src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  border: $card-border;
```

Comment:
The variable `$card-border` is used here, but it's not defined within this file. Ensure that it is defined in a file that is imported before this one or within this file itself.